### PR TITLE
Add alternate sources for versioning metadata

### PR DESCRIPTION
**Pull request by jboursiquot**
We look in the environment as well as vars set at build time.

Works in tendem with https://github.com/AchievementNetwork/sas/pull/38.

### CODE REVIEW

File: vascoClient/statusHelpers.go [hunk 1]
```diff
@@ -83,10 +83,10 @@ func (vc *VascoClient) GetStatus() Status {
 		vc.status.IP = ip
 	}
 	vc.status.Uptime = time.Since(vc.status.StartTime).String()
```
Comment (kentquirk) [inline vascoClient/statusHelpers.go]: I don't know why this code was originally duplicated -- I think I must have been interrupted in the middle of an edit. But given that GetStatus calls FillStatus, there's no need for it to then re-fetch all that information. Can you delete everything but the first and last lines from this function, please?

File: vascoClient/vascoClient.go [hunk 2]
```diff
@@ -29,6 +29,36 @@ import (
 // These will be set by the linker when built using the standard tools
 var SourceRevision string = "Not set"
 var SourceDeployTag string = "Not set"
+var SourceDeployType string = "Not set"
+var SourceConfigVersion string = "Not set"
+
+func getRevision() string {
+	if len(os.Getenv("REVISION")) > 0 {
```
Comment (kentquirk) [inline vascoClient/vascoClient.go]: Everywhere else I've done this, I have checked for equality to the empty string. `os.Getenv("REVISION")==""` For consistency, do you mind changing it? I was wondering if there was an efficiency benefit, so I did a quick benchmark to compare the two, and on my machine they were both around 1 nSec/op, and within 10% of each other. So it's more about style.

Comment (jboursiquot) [inline vascoClient/vascoClient.go]: Yep, that's cool. Will update.

File: vascoClient/statusHelpers.go (hunk 1 above)
Comment (jboursiquot) [inline vascoClient/statusHelpers.go]: Sure thing.

**Comment by jboursiquot** [comment]
@kentquirk ready when you are.

**Pull request merged**